OoyalaSkinSampleApp/OoyalaSkinSampleApp.xcodeproj/project.pbxproj

```diff
@@ -441,8 +441,7 @@
 				ORGANIZATIONNAME = "Ooyala, Inc";
 				TargetAttributes = {
 					13B07F861A680F5B00A75B9A = {
-						DevelopmentTeam = 556Y9XWD3N;
-						ProvisioningStyle = Manual;
+						ProvisioningStyle = Automatic;
```

Comment:
Remove the changes you did in this file for the OoyalaSkinSampleApp

Reply:
Done